sdks/python/apache_beam/transforms/environments.py

```diff
@@ -111,11 +111,12 @@ class Environment(object):
   _known_urns = {}  # type: Dict[str, Tuple[Optional[type], ConstructorFn]]
   _urn_to_env_cls = {}  # type: Dict[str, type]
 
-  def __init__(self,
+  def __init__(
```

Comment:
Again, the huge number of irrelevent whitespace changes make it hard to see what the actual (subtle) relevant bits are.